src/app/shared/feature-resolver/feature-resolver.service.ts

```diff
@@ -9,7 +9,7 @@ export class FeatureResolverService extends FeatureStateResolver {
       case ApplicationFeature.PageTimeRange:
         return of(false as T);
       case ApplicationFeature.FeatureDefaultTimeRange:
-        return of({} as T);
+        return of(({} as unknown) as T);
```

Comment:
nit - don't cast to unknown. What was the issue? Either `FeatureFlagValue` is missing the type, or the fact that `{}` is empty means it's not matching the dictionary type in which case we'd rather cast it to something like `{} as Dictionary<string> & T`

Reply:
it was a tslint issue `no-object-literal-type-assertion`

Reply:
object literals can only be cast to `any` or `unknown`

Reply:
You can either split it into declaration and usage or just ignore the lint rule. We don't want to change the casts to be less accurate to satisfy the linter, it's better to turn off the linter if we consider this an exception.

```
        const value = {};
        
        return of(value as T);
```
or
```
        // tslint:disable-next-line: no-object-literal-type-assertion
        return of({} as T);
```
would both be preferable.